src/som/interpreter/nodes/SOMNode.java

```diff
@@ -33,9 +33,11 @@
 
 @TypeSystemReference(Types.class)
 public abstract class SOMNode extends Node implements MateNode {
+  protected final SourceSection sourceSection;
 
   public SOMNode(final SourceSection sourceSection) {
-    super(sourceSection);
+    super();
```

Comment:
This is just because in a Truffle update, Source Sections were taken out from the Node class and move their responsibility to guest languages. In standard SOM this change has been done while ago i think but not in Mate (at least in the master branch).